packages/components/tooltip/README.mdx

```diff
@@ -40,4 +44,18 @@ Tooltips are very useful to communicate extra information related to an element
 
 ## Accessibility
 
-missing
+- Do not put essential information in `Tooltip` component
+- Add unique `id` property to the tooltip, so appropriate a11y attributes could be used.
+
+```jsx
+import { Tooltip } from '@contentful/f36-components';
+<Tooltip placement="top" id="tooltip" content="Hi I am a Tooltip">
+  <TextLink>Hover me</TextLink>
+</Tooltip>;
+```
+
+## Props
+
+import { Props } from '@contentful/f36-docs-utils';
+
+<Props of="Tooltip" />
```

Comment:
This needs the `typescript` YAML frontmatter key to work. An example from Button: https://github.com/contentful/forma-36/blob/forma-v4/packages/components/button/README.mdx?plain=1#L8